tests/test-recipes/metadata/numpy_build_run/run_test.bat

```diff
@@ -1,5 +1,7 @@
-@echo on
+:: show list
 conda list -p "%PREFIX%" --canonical
 if errorlevel 1 exit 1
-conda list -p "%PREFIX%" --canonical | grep "conda-build-test-numpy-build-run-1\.0-py..h......._0"
```

Comment:
This glob pattern is problematic once we start testing with Python 3.10. Notice the `py..h` pattern near the end, this will successfully match py38 and py39 but not py310 or py311. Replaced this explicit pattern with a more fuzzy `py.*` pattern.

Reply:
this is not within the scope of this PR.. I'll make the change for it in the next ticket..

Reply:
this is a side effect of changing the Python versions being tested